lms/views/content_item_selection.py

```diff
@@ -8,6 +13,30 @@
 from lms.views.decorators import create_h_user
 
 
+def _check_params(lti_params):
+    except_msg = 'Required LTI data param for content item selection missing: '
+
+    try:
```

Comment:
As the `try/except` blocks here all do the same general thing—try to access a given required param, raise a predictable Exception on `KeyError`...I wonder if you might be able to factor the `_check_params` function such that it takes a list of required parameter names as an argument (e.g. `_check_params(required_params)`) and then it could condense the function body down to:

```
except_msg = _('Required LTI data param for content item selection missing: ')

for required_param in required_params:
    try:
        lti_params[required_param]
    except KeyError:
        raise MissingLTIContentItemParamError(except_msg + required_param)
```